Doc/library/sqlite3.rst

```diff
@@ -47,83 +47,104 @@ This document includes four main sections:
       PEP written by Marc-André Lemburg.
 
 
+.. We use the following practises for SQL code:
+.. - UPPERCASE for keywords
+.. - lowercase for schema
+.. - single quotes for string literals
+.. - singular for table names
+.. - if needed, use double quotes for table and column names
+
 .. _sqlite3-tutorial:
 
 Tutorial
 --------
 
-To use the module, start by creating a :class:`Connection` object that
-represents the database.  Here the data will be stored in the
-:file:`example.db` file::
+In this tutorial, you will learn the basics of the :mod:`!sqlite3` API
+by creating an on-disk database :file:`python.db`
+for managing Python release metadata.
+A fundamental understanding of database concepts,
+like `transactions`_ and `cursors`_, is assumed.
+
+To start working with an SQLite database,
+you must first open a connection to it.
+Call the :func:`sqlite3.connect` function
+to create a :class:`Connection` to the database :file:`python.db`::
 
    import sqlite3
-   con = sqlite3.connect('example.db')
+   con = sqlite3.connect("python.db")
 
-The special path name ``:memory:`` can be provided to create a temporary
-database in RAM.
+SQLite will implicitly create the database file :file:`python.db`
+in the current working directory if it does not exist.
+The returned :class:`Connection` object ``con``
+represents our connection to the on-disk database.
 
-Once a :class:`Connection` has been established, create a :class:`Cursor` object
-and call its :meth:`~Cursor.execute` method to perform SQL commands::
+In order to execute SQL statements and fetch results from SQL queries,
+we need to use a database cursor.
```

Comment:
```suggestion
use a database cursor.
```

Most of the tutorial uses second person singular but this uses first person plural, so we (heh) should be consistent here—this is probably my fault since I do this a lot myself. Also, this allows us to be more concise and direct here.

Reply:
Imperative mode, ftw.